.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
Plase remove this file and add .settings to .gitignore

Reply:
Fixed.